### ISSUE TITLE
Element.bind is not function (while using ui-sref)

### ISSUE THREAD

**Issue by mehowp** [angular-ui/ui-router#2743]
Hello.

I'm using angular 1.5.5 and newest ui-router (0.2.18). I have successfuly configured my angularjs app, states works fine but problem occurs when I'm trying to use ui-sref directive:

```
Error: element.bind is not a function
$StateRefDirective/<.link@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:8499:7
invokeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:18486:9
nodeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17885:11
compositeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17189:13
compositeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17192:13
compositeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17192:13
compositeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17192:13
publicLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17069:30
browserifyShim/</ngIncludeFillContentDirective</<.link@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:34659:9
invokeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:18486:9
nodeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17885:11
compositeLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17189:13
publicLinkFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17069:30
lazyCompilation@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17407:16
boundTranscludeFn@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17206:16
controllersBoundTransclude@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:17935:20
ngIncludeWatchAction/<@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:34605:27
processQueue@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:24753:28
scheduleProcessQueue/<@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:24769:27
$RootScopeProvider/this.$get</Scope.prototype.$eval@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:26021:16
$RootScopeProvider/this.$get</Scope.prototype.$digest@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:25837:15
$RootScopeProvider/this.$get</Scope.prototype.$apply@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:26129:13
done@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:20364:36
completeRequest@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:20570:7
requestLoaded@http://localhost/iw/wp-content/themes/bbstudio-michal/resources/includes/js/angular.js:20503:9
```

`<a class="navbar-brand" ui-sref="header">`

Does anyone know why this happen and how to fix it?

**Comment by christopherthielen** [comment]
I haven't seen this occur. Are you able to reproduce this in a plunk http://bit.ly/UIR-Plunk ?  What browsers is this happening in?

I thought that the `element.bind()` code was introduced in 0.3.0, not in 0.2.18

**Comment by eddiemonge** [comment]
Closing due to inactivity

**Comment by oznu** [comment]
I experienced the same issue. The root cause for me was the jQuery library I had included. I swapped from ```jquery.slim.js``` back to ```jquery.js``` and the errors went away.

**Comment by Masireddy** [comment]
I faced the same issue and it got resolved by removing jqery.slim.js from my index.html page

**Comment by sbaechler** [comment]
Duplicate of #3035 
Resolved in #3036 

Released in 0.3.2
